serving/docs/python.md

```diff
@@ -0,0 +1,111 @@
+# DJLServing Python Mode
+
+## Overview
+
+DJL Serving is a high-performance serving system for deep learning models. DJL Serving supports Python models, Java models, as well as binary models.
+
+This document walks through how to serve Python based model with DJL Serving.
+
+## Installation
+
+Install `djl_python` toolkit:
+
+```
+cd djl-serving/engines/python/setup
+pip install -U -e .
+```
+
+## Model Implementation
+
+You can run inference with DJL Serving against the Python EngineProvider.
+
+To get started, implement a python source file named `model.py` as the entry point. DJL Serving will run your request by invoking a `handle` function that you provide. The `handle` function should have the following signature:
+
+```
+def handle(inputs: Input)
+```
+
+If there are other packages you want to use with your script, you can include a `requirements.txt` file in the same directory with your model file to install other dependencies at runtime.
+
+A `requirements.txt` file is a text file that contains a list of items that are installed by using pip install. You can also specify the version of an item to install.
+
+## Serving Example
+
+Let's run an example where we load a model and run inference using the REST API.
+
+### Step 1: Start Server
+
+First, start the server and load a model at startup. We will use the resnet18 model as an example. To get this model, first clone the djl-demo repo.
+
+```
+mkdir -p /tmp/djl
```

Comment:
avoid hard code path in example:

git clone https://github.com/deepjavalibrary/djl-demo.git
cd djl-demo